### ISSUE TITLE
Header Component

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/www.greenwoodjs.dev#26]
## Summary

Will want to create a component for the global site header, with logo on the left and links / socials on the right.  Needs to be done after #23 

> Needs to be done after #23 

![Screenshot 2024-04-17 at 9 16 05 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/72de93f6-f5a8-4f6d-b756-c231c36b3150)
![Screenshot 2024-04-17 at 9 16 13 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/bdfe0055-fb71-4ce1-9b0d-199f9f7c09b2)
![Screenshot 2024-04-17 at 9 16 17 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/529d64d9-dec7-4e2b-9c2a-6c3c94e6cfe4)

## Details
1. [x] Will need a _src/components/header/header.js_ component file
1. [x] Will want a Storybook file
1. [x] Will want a spec file to test basic content output
1. [x] Since this one is interactive on mobile breakpoints, we should use declarative shadow DOM in our implementation
1. [x] (nice to have) what could be a JavaScript disabled fallback for this look like?

----

## Open Items
1. [x] Need a canonical figma file / screenshot for reference here
    - Figma shows different treatments for home page vs docs / guides pages
    - Need social links in the header - Twitter, GitHub, Discord
    - Social links and logs disappear in mobile?
1. [x] Will need / want the social and project logos as SVGs - hoping to get them all in #36

**Comment by thescientist13** [comment]
Some references
- https://github.com/AnalogStudiosRI/www.blissri.com/tree/main/src/components/header
- https://github.com/AnalogStudiosRI/www.blissri.com/tree/main/src/components/navigation
